filelog_unix.go

```diff
@@ -0,0 +1,5 @@
+// +build !windows
+
+package log4go
+
+const FILELOG_ARCHIVE_REGEX = "\\.[0-9]{4}-[0-9]{2}-[0-9]{2}(\\.[0-9]{4})?(\\.gz)?$"
```

Comment:
I'm surprised this regex covers the whole range of logging options users can configure - can't users have log files with just numbered extensions and no dates? Or do we only support `<log name>.<date>.<optional suffix if file rotates within the day>.<optional compressed extension>`?